ansible/group_vars/all

```diff
@@ -0,0 +1,2 @@
+keys:
+  - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCntA+4D8gAXocrFrC1MYhAyT2QzS5Qpx2emsg/hhjrxYVizmDQFV0BpqSPXDGEEFAcZNIQay/0i3u6JXoLnoB5xHJYHAk7xlRVKsX3T74MXFxucNQK5ZBlMIDGd2+T1JnqLM4lJ3Qo76q05ljloURFCkkchTsczUiKxKei42RI7bBVAjvZTREqalZQ+Il4j8h7G7Uy3CVkyLxYYQka4vFPAUUGov4BTu/WOtbmmcZ9pkQFIkoGSqXlRznurpgOE5gf+AiPytHGmyvVOz/G2O0Bpha0pgZtxJVVZnEvi8PqLQCHX1qM/GoW50UYIVnKdyhlS6Mg9DByETlC86N6Uj5d lordaro@Apollo"
```

Comment:
Could you please add:
`ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDAhcPjy6TLOEPhhCfW6ap3rGgJWxUHanrRb/7pgFTM1uDsu/6qWieP0I6ivVVs4GByJgqqPHfzscLuoGuZoJ/CoskXRaZCzeextCRFHWtiiwLR1HB0Z954oHi+u8e4avYEdrsIteJva+bPiW8pjaKq/3nUyr4sQZzRVdsmU9kM3ZzXFt7lQvm75Mphoeu3As2BbAAs9SJzC/P2Odu5jSINHyOpjp2mzz0NRcE25FQzjGOrkpuGNqdKM7NgkXSQxZ/ozbp2cNW2gF+fEDtI6+LbwkhkqYbuKJWw2zuuPTZ3UFv95RY7/M1nUH9SF8Ci/IY0h1g3QU7YOa/ZtoktQs/abyfjIYRyw28jC6V3DOGEifd3FlHpdVRN3H9aNNK207a+0eKKvM7C0ZWPOJQAJSrCndj6QlrKnpHNG5a8bDAwqdtDQ7pK1YJkhI1W9HNUJ6cf9E6JXEphdoaBxE2YaFPkSzjzIzTiTIehHN2VRfCl1DKNcKy2XpbwMYE+6/PZf2U= truebrain@truebrain`
and deploy this? Tnx :)